python/scraper.py

```diff
@@ -23,12 +23,22 @@ def log(msg, level=xbmc.LOGDEBUG):
 def get_tmdb_scraper(settings):
     language = settings.getSettingString('language')
     certcountry = settings.getSettingString('tmdbcertcountry')
-    return TMDBMovieScraper(ADDON_SETTINGS, language, certcountry)
+    search_language = settings.getSettingString('searchlanguage')
+    return TMDBMovieScraper(ADDON_SETTINGS, language, search_language, certcountry)
 
 def search_for_movie(title, year, handle, settings):
     log("Find movie with title '{title}' from year '{year}'".format(title=title, year=year), xbmc.LOGINFO)
     title = _strip_trailing_article(title)
     search_results = get_tmdb_scraper(settings).search(title, year)
```

Comment:
Save the scraper instance returned from `get_tmdb_scraper` here, and then reuse it for the following searches.